components/channel_header/index.js

```diff
@@ -47,12 +47,14 @@ import {isFileAttachmentsEnabled} from 'utils/file_utils';
 
 import ChannelHeader from './channel_header';
 
+const EMPTY_CHANNEL = {};
+
```

Comment:
that change is surely not needed as long as the variable is only used once

Reply:
see https://github.com/mattermost/mattermost-webapp/pull/9808#discussion_r807094640
so until we get the request back from server about the channel or until store is populated we dont have the reference of the object `EMPTY_CHANNEL` to change